static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -498,4 +526,15 @@ const StyledSearchBar = styled(SearchBar)`
   }
 `;
 
+const StyledWarningIcon = styled(IconWarning)`
+  color: ${p => p.theme.alert.warning.iconColor};
+  height: ${p => p.theme.iconSizes.sm};
+  width: ${p => p.theme.iconSizes.sm};
+  display: block;
+
+  &:hover {
+    color: ${p => p.theme.alert.warning.iconHoverColor};
```

Comment:
Does it need a hover?

(I think you can probably delete this styled component)

Reply:
I needed to center it so I still needed to apply` display` attribute unfortunately